server/routes/components.ts

```diff
@@ -96,20 +96,24 @@ export default function routes({ serviceCatalogueService, redisService, alertsSe
     const environmentName = getEnvironmentName(req)
 
     const component = await serviceCatalogueService.getComponent({ componentName })
-    const environments = component.environments?.filter(environment => environment.name === environmentName)
+    const filteredEnvironment = component.envs?.data?.filter(
+      environment => environment.attributes.name === environmentName,
+    )
     const activeAgencies =
-      environments.length === 0 ? '' : formatActiveAgencies(environments[0].active_agencies as Array<string>)
+      filteredEnvironment.length === 0
+        ? ''
+        : formatActiveAgencies(filteredEnvironment[0].attributes.active_agencies as Array<string>)
     const allowList = new Map()
 
-    if (environments[0].ip_allow_list && environments[0].ip_allow_list_enabled) {
-      const ipAllowListFiles = Object.keys(environments[0].ip_allow_list)
+    if (filteredEnvironment[0].attributes.ip_allow_list && filteredEnvironment[0].attributes.ip_allow_list_enabled) {
```

Comment:
I would assign `filteredEnvironment[0].attributes` to a new variable as you're using it a lot, making the code look more complicated than it is, same across other areas that are doing this.

Reply:
Maybe `const attributes = filteredEnvironment[0].attributes` 🤷‍♂️